src/plugin.js

```diff
@@ -819,7 +819,7 @@ for (let i = 0; i < _readOnly.length; i++) {
  *          - False otherwise.
  */
 Flash.isSupported = function() {
```

Comment:
We should update the doc comment for this method to say it always returns true and why we're doing so. Let me know if you need help with wording/reasoning.

Reply:
Does this sound fine : 
/**
 * Check if the Flash tech is currently supported.
 *
 * @return {boolean}
 *          - True always as the latest versions of Chrome and Firefox automatically 
 *            blocks flash by default.
 */